worker-pools.yml

```diff
@@ -1581,6 +1581,31 @@ pools:
             - <<: *persistent-disk
               diskSizeGb: 60
           machine_type: c2-standard-4
+  - pool_id: '{pool-group}/t-linux-2404-x11-relsre'
+    description: Workers for x11 on Ubuntu 24.04. Used by RelSRE to qualify images.
+    owner: jmoss@mozilla.com
+    email_on_error: true
```

Comment:
We should move this to a google group.

Reply:
Yeah I think that was for testing. I'll update it.